pkg/commands/compute/build_test.go

```diff
@@ -532,3 +533,152 @@ func TestBuildJavaScript(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildOther(t *testing.T) {
+	args := testutil.Args
+	if os.Getenv("TEST_COMPUTE_BUILD") == "" {
+		t.Log("skipping test")
+		t.Skip("Set TEST_COMPUTE_BUILD to run this test")
+	}
+
+	// We're going to chdir to a build environment,
+	// so save the PWD to return to, afterwards.
+	pwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create test environment
+	//
+	// NOTE: Our only requirement is that there be a bin directory. The custom
+	// build script we're using in the test is not going to use any files in the
+	// directory (the script will just `echo` a message).
+	rootdir := testutil.NewEnv(testutil.EnvOpts{
+		T: t,
+		Write: []testutil.FileIO{
+			{Src: "mock content", Dst: "bin/testfile"},
+		},
+	})
+	defer os.RemoveAll(rootdir)
+
+	// Before running the test, chdir into the build environment.
+	// When we're done, chdir back to our original location.
+	// This is so we can reliably copy the testdata/ fixtures.
+	if err := os.Chdir(rootdir); err != nil {
+		t.Fatal(err)
+	}
+	defer os.Chdir(pwd)
+
+	for _, testcase := range []struct {
+		args                 []string
+		dontWantOutput       []string
+		fastlyManifest       string
+		name                 string
+		stdin                string
+		wantError            string
+		wantOutput           []string
+		wantRemediationError string
+	}{
+		{
+			name: "no custom build",
+			args: args("compute build --language other"),
+			fastlyManifest: `
+			manifest_version = 2
+			name = "test"
+			language = "other"`,
+			dontWantOutput: []string{
+				"This project has a custom build script defined in the fastly.toml manifest",
+			},
+			wantError:            "error reading custom build instructions from fastly.toml manifest",
+			wantRemediationError: "Add a [scripts.build] setting for your custom build process",
+		},
```

Comment:
Could we also add a test for a non-other language?

Reply:
done